src/cpptools.ts

```diff
@@ -117,7 +116,7 @@ export function parseCompileFlags(cptVersion: cpt.Version, args: string[], lang?
   const can_use_gnu_std = (cptVersion >= cpt.Version.v4);
   const iter = args[Symbol.iterator]();
   const extraDefinitions: string[] = [];
-  let standard: StandardVersion = (lang === 'C') ? 'c11' : 'c++17';
+  let standard: StandardVersion | undefined = undefined;
```

Comment:
remove " = undefined", declaring with " | undefined" and not initializing means initialized with undefined. The CI fails with a warning about this also.